### PR TITLE
fix(auth): Defines missing `DecodedIdToken` types.

### PR DESCRIPTION
**Pull request by bojeil-google**
Fixes https://github.com/firebase/firebase-admin-node/issues/846

RELEASE NOTE: Exposed `email`, `email_verified`, `phone_number` and `picture` fields from the `DecodedIdToken` type.

### CODE REVIEW

File: src/auth.d.ts [hunk 1]
```diff
@@ -420,6 +420,17 @@ export namespace admin.auth {
      */
     auth_time: number;
 
+    /**
+     * The email corresponding to the user who the ID token belonged to, if available.
```
Comment (egilmorez) [inline src/auth.d.ts:424]: Suggest "The email of the user to whom the ID token belonged, if available."

Comment (egilmorez) [inline src/auth.d.ts:424]: The email of the user to whom the ID token belonged, if available.

File: src/auth.d.ts [hunk 2]
```diff
@@ -420,6 +420,17 @@ export namespace admin.auth {
...
+     */
+    email?: string;
+
+    /**
+     * Whether the email corresponding to the user who the ID token belonged to is
```
Comment (egilmorez) [inline src/auth.d.ts:429]: Suggest "Whether or not the email of the user to whom the ID token belonged is verified, provided the user has an email."

Are the valid values true and false?

File: src/auth.d.ts [hunk 3]
```diff
@@ -494,6 +505,18 @@ export namespace admin.auth {
      */
     iss: string;
 
+    /**
+     * The phone number corresponding to the user who the ID token belonged to,
```
Comment (egilmorez) [inline src/auth.d.ts:509]: Suggest "The phone number of the user to whom the ID token belonged, if available."

Or -- can these be present tense "belongs?"  That would be preferable if it is accurate.

File: src/auth.d.ts [hunk 4]
```diff
@@ -494,6 +505,18 @@ export namespace admin.auth {
...
+     * if available.
+     */
+    phone_number?: string;
+
+    /**
+     * The photo URL corresponding to the user who the ID token belonged to,
```
Comment (egilmorez) [inline src/auth.d.ts:515]: Suggest "The photo URL for the user to whom the ID token belonged..."

File: src/auth.d.ts (hunk 1 above)
Comment (bojeil-google) [reply src/auth.d.ts]: Done

File: src/auth.d.ts (hunk 2 above)
Comment (bojeil-google) [reply src/auth.d.ts]: Yes, true or false. Done.

File: src/auth.d.ts (hunk 3 above)
Comment (bojeil-google) [reply src/auth.d.ts]: Done on both.

File: src/auth.d.ts (hunk 4 above)
Comment (bojeil-google) [reply src/auth.d.ts]: done

**Pull request merged**